CIP-0000.md

```diff
@@ -0,0 +1,195 @@
+* An **Informational CIP** describes a Celo design issue, or provides general guidelines or information to the Celo community, but does not propose a new feature. Informational CIPs do not necessarily represent Celo community consensus or a recommendation, so users and implementers are free to ignore Informational CIPs or follow their advice.
+
+It is highly recommended that a single CIP contain a single key proposal or new idea.
+
+A CIP must meet certain minimum criteria. It must be a clear and complete description of the proposed enhancement. The enhancement must represent a net improvement as to be determined by the CIP editors during the PR review process. For more information on the CIP Editor role, see the appropriate section below.
+
+
+# CIP Workflow
+## Shepherding a CIP
+Parties involved in the process are you, the champion or _CIP author_, the CIP editors, and the Celo Core Developers (core developers are the developers working on the Celo Platform).
+
+Before you begin writing a formal CIP, you should vet your idea. Ask the Celo community first if an idea is original to avoid wasting time on something that will be be rejected based on prior research. It is thus recommended to open a discussion thread on Issues section of the CIP process to do this, but you can also use the [Celo Discord](https://discord.gg/YHf9Vs4) and the [Celo Forum](https://forum.celo.org) and engage with the community about your idea.
+
+In addition to making sure your idea is original, it will be your role as the author to make your idea clear to reviewers and interested parties, as well as inviting editors, developers and community to give feedback on the aforementioned channels. You should try and gauge whether the interest in your CIP is commensurate with both the work involved in implementing it and how many parties will have to conform to it. For example, the work required to implement a Ring 0/Ring 1 CIP will be much greater than for a Ring 2/ Ring 3 CRC and the CIP will need to be reviewed carefully by the Celo community. Negative community feedback will be taken into consideration and may prevent your CIP from moving past the Draft stage.
+
+Building consensus with the Celo community must adhere to the [Celo Code of Conduct](https://celo.org/code-of-conduct). Further, your proposal is much more likely to build consensus if you it aligns with the [Celo Community Tenets](https://celo.org/community) and furthers the Celo project's mission.
+
+## Ring 0 and Ring 1 CIPs
+For Ring 0-1 CIPs, given that they require Celo’s client implementation to be considered **Final** (see “CIPs Process” below), the CIP author will need to either provide an implementation for the celo-blockchain client or convince clients to implement your CIP.
```

Comment:
nit: convince core devs?

Reply:
Generally I find it confusing every where we refer to "clients", would suggest we refrain from implying multiple Celo clients until others exist